### DISCUSSION TITLE
Multi-Dataset Sampling for Speaker Recognition Fine-tuning

### DISCUSSION THREAD

**Discussion by ap48923** [NVIDIA-NeMo/NeMo#3292] [Q&A]
I am fine-tuning an ECAPA-TDNN model for speaker verification using [this](https://github.com/NVIDIA/NeMo/blob/main/examples/speaker_tasks/recognition/speaker_reco_finetune.py) provided script in the speaker_tasks examples directory. 

From the `00_NeMo_Primer.ipynb` tutorial I see that I can include multiple datasets in `manifest_path` by entering a comma-separated list of manifest file paths. Is there a way for me to specify a sampling approach for each dataset during training? For example, given that I have 2 datasets, I would like to ensure that each training batch contains 50% from each dataset.

**Comment by VahidooX** [comment]
I don't think we have such a feature now. @nithinraok may know a solution for this.
